page-objects/checkoutPage.ts

```diff
@@ -1,13 +1,31 @@
 import {Locator, Page} from '@playwright/test'
 import { HelperBase } from './helperBase'
-import { ShippingRate } from '../e2e/enums'
+import { ShippingRate } from '../helper/enums'
 
 export class CheckoutPage extends HelperBase{
 
     constructor(page: Page){
         super(page)
     }
 
+    /**
+     * Asynchronously completes the checkout process as a guest user (not signed in). Handles entering 
+     * shipping details and selecting a shipping rate.
+     *
+     * @param {string} email - The email address for the order.
+     * @param {string} firstName - The billing address first name.
+     * @param {string} lastName - The billing address last name.
+     * @param {string} company - The billing address company name (optional).
+     * @param {string} streetAddress1 - Street Address Line 1.
+     * @param {string} streetAddress2 - Street Address Line 2 (optional).
+     * @param {string} streetAddress3 - Street Address Line 3 (optional).
+     * @param {string} city - The billing address city.
+     * @param {string} zip - The billing address zip/postal code.
+     * @param {string} country - The billing address country.
+     * @param {string} phoneNumber - The billing address phone number.
+     * @param {string} stateProvince - The billing address state/province.
+     * @param {ShippingRate} shippingRate - An enum value specifying the shipping rate type (e.g., Fixed, Table Rate).
+     */
     async checkoutWhileNotSignedIn(email: string, firstName: string, lastName: string, company: string, streetAddress1: string, streetAddress2: string,
```

Comment:
Do we need to use an implicit wait In checkoutWhileNotSignedIn or is there another way?

Reply:
I haven't used this function for my tests so far (I made it while we were preparing the project structure). So if it is okay, if there is a need to use, it I will correct this.